sysdig/testfiles/ECSinput.json

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
This is great! I think a few more test files would be useful but that can be added in the future, since this looks like it's exercising the paths we're worried about